pkg/openstack/loadbalancer.go

```diff
@@ -605,6 +606,32 @@ func (lbaas *LbaasV2) getLoadBalancerLegacyName(_ context.Context, _ string, ser
 	return cloudprovider.DefaultLoadBalancerName(service)
 }
 
+// this function gets the listeners tags from the service annotation and   
+func (lbaas *LbaasV2) customLoadBalanceListenerTag(service *corev1.Service) ([]string, error) {
```

Comment:
```suggestion
func (lbaas *LbaasV2) customLoadBalancerListenerTag(service *corev1.Service) ([]string, error) {
```

This belongs to an LB object

Reply:
I’m fully aware and thank you for pointing out the almost invisible to see typo

Reply:
i actually took your comment in good faith and only pointed that it'd be great to not repeat others comment because of bulky messages in the PR is kinda confusing and repetitive ones are overwhelming 

also, The typo is almost invisible and even the previous two reviewer  @pierreprinetti and @kayrus missed it
these two are almost impossible to notice the difference `customLoadBalanceListenerTag` and `customLoadBalancerListenerTag`

so i am good with you and you probably took my reviews out of context
i'm sorry if you feel different